.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
```

Comment:
```suggestion
name: Test
```

This whole thing is just the express tests, lets just rename the file and the title.

Reply:
I had not plan with the naming.  Let's change it.

Reply:
I've made this change (changing the file name)